src/Services/QueuedJobService.php

```diff
@@ -1328,9 +1540,9 @@ public function getJobListFilter($type = null, $includeUpUntil = 0)
      *
      * @param string $queue
      */
-    public function runQueue($queue)
+    public function runQueue($queue): void
```

Comment:
Method signatures must not  be changed outside of a major release. Please revert this change.